tests/phpunit/elementor/modules/site-navigation/data/recent/test-controller.php

```diff
@@ -233,6 +234,67 @@ public function test_create_items() {
 		$this->assertStringContainsString( $expected_edit_url, $edit_url );
 	}
 
+
+	/**
+	 * Test invalid post id - duplicate post
+	 */
+	public function test_create_items__invalid_post_id() {
+		// Arrange.
+		$this->act_as_editor();
+
+		// Act.
+		$params = [
+			'post_id' => 'test',
+			'title' => 'test page',
+		];
+
+		$response = $this->send_request( 'POST', self::DUPLICATE_POST_ENDPOINT, $params );
+		// Assert.
+		$this->assertEquals( 400, $response->get_status() );
+	}
+
+	public function test_create_items_duplicate_post() {
+		// Arrange.
+		$this->act_as_editor();
+
+		// Act.
+		$post = self::factory()->post->create( [
+			'post_type' => 'post',
+			'post_title' => 'Test Post',
+			'post_status' => 'publish',
+		] );
+
+		$params = [
+			'post_id' => $post,
+		];
+
+		$response = $this->send_request( 'POST', self::DUPLICATE_POST_ENDPOINT, $params );
+
+		// Assert.
+		$this->assertEquals( 200, $response->get_status() );
+
+		$duplicated_post_id = $response->get_data()['post_id'];
+
+		$this->assertTrue( $duplicated_post_id > $post );
+	}
+	
+	public function test_create_items_duplicate_post__forbidden() {
+		// Arrange.
+		$this->act_as_subscriber();
+
+		// Act.
+		$params = [
+			'post_id' => 'test',
```

Comment:
send a real id to mock the right conditions

Reply:
This test checks wrong post id, so string is sent...

Reply:
This is the test for forbidden role